release/release_logs/2.0.0/scalability/single_node.json

```diff
@@ -11,30 +11,30 @@
         {
             "perf_metric_name": "10000_args_time",
             "perf_metric_type": "LATENCY",
-            "perf_metric_value": 17.831142194999984
+            "perf_metric_value": 17.26158977700004
         },
         {
             "perf_metric_name": "3000_returns_time",
             "perf_metric_type": "LATENCY",
-            "perf_metric_value": 6.268522376000021
+            "perf_metric_value": 6.01299958300001
         },
         {
             "perf_metric_name": "10000_get_time",
             "perf_metric_type": "LATENCY",
-            "perf_metric_value": 26.833145973
+            "perf_metric_value": 26.108915361000015
         },
         {
             "perf_metric_name": "1000000_queued_time",
             "perf_metric_type": "LATENCY",
-            "perf_metric_value": 178.32711092900007
+            "perf_metric_value": 204.12705361400003
```

Comment:
same tendency here (single client regression)

Reply:
Right - https://b534fd88.us1a.app.preset.io/superset/explore/p/wxr7XGpL2JQ/

This seems to be bad. I am looking into this now.

Reply:
This could be related to `stage_2_avg_iteration_time` as well. 

And potentially https://github.com/ray-project/ray/commit/deaae4ce628e5e0313cf8f0874178919b99fd51f as we are now tracking task states transitions in the task execution critical paths.

Reply:
Given the regression trend (slow) + the regression absolute value (<10%), we will not make it release blocking, but follow up in https://github.com/ray-project/ray/issues/29766